bot/cogs/wolfram.py

```diff
@@ -0,0 +1,65 @@
+import aiohttp
+import discord
+import os
+from io import BytesIO
+from urllib import parse
+from discord.ext import commands
+from bot.settings import APPID
+
+
+class Wolfram(commands.Cog):
+    """command for wolfram search"""
+    def __init__(self, bot):
+        self.bot = bot
+        self.query = "http://api.wolframalpha.com/v2/{request}?{data}"
+        
+
+    # wolfram command, takes in a search and gives the result
+    # PS, I didn't totally steal it from seasonal bot
+    @commands.command(brief="Takes in a wolfram search and displays the result", usage=".wolfram [query]")
+    async def wolfram(self, ctx, *, query):
+
+        url_str = parse.urlencode({
+            "i": query,
+            "appid": APPID,
+        })
+
+        query_final = self.query.format(request="simple", data=url_str)
+
+        async with ctx.channel.typing():
+            async with aiohttp.ClientSession() as cs:
+                async with cs.get(query_final) as response:
```

Comment:
`self.bot` has an attribute `session`, which you can use instead of creating a new `aiohttp.ClientSession`.

Everything after this should be brought back an indent.

```suggestion
            async with self.bot.session.get(query_final) as response:
```